_faq/installation_and_deployment/update-cron.md

```diff
@@ -0,0 +1,13 @@
+---
+title: How to update the Cron Job?
+order: 8
+---
+{::options auto_ids="false" /}
+
+Forseti is installed in the `ubuntu` account. To update the cron job you need to SSH into the server VM and switch to 
```

Comment:
Please change the first sentence to this:
```
Forseti is installed with the `ubuntu` user account on the server VM.
```

Reply:
Done